### ISSUE TITLE
The colour picker seems to detect colours from a wrong place on the screen

### ISSUE THREAD

**Issue by micnekr** [flameshot-org/flameshot#1714]
**Flameshot version**
Flameshot v0.10.0
Compiled with Qt 5.15.2

**Describe the bug**
When using the "grab color" option in "tool settings", the tool seems to detect a color from a different portion of the screen, for example here: 
  
![2021-07-19_11-55](https://user-images.githubusercontent.com/44928743/126150268-1c8afd31-03b7-4a6b-80bb-c239c6436399.png)

If I move the color picker to the location of the pink cross, it seems to detect the color at the location of the green cross. If I move the color picker around, it shows colors adjacent to the green cross.

**To Reproduce**
Run `flameshot gui` and select "tool settings", press the "grap color" button and then move the cursor somewhere on the screen.

**Expected behavior**
The color selected should be the same as the color under the cursor

**System Information**
Arch Linux, KDE plasma.

**Comment by mmahmoudian** [comment]
Please fill the bug report with extra attention to details so that we can better understand the situation and move towards fixing it. You didn't mention if you are on Wayland or Xorg. If you don't know how to find that information, please check out the following:

https://flameshot.org/guide/issue-reporting/

**Comment by borgmanJeremy** [comment]
My gut says this is a fractional scaling issue since this works for me

**Comment by micnekr** [comment]
> Please fill the bug report with extra attention to details so that we can better understand the situation and move towards fixing it. You didn't mention if you are on Wayland or Xorg. If you don't know how to find that information, please check out the following:
> 
> https://flameshot.org/guide/issue-reporting/

Yes, sorry, I did not see that this information is needed. I am using X. Also, it does seem to be an issue with scaling because changing "global scale" in plasma to 100% made everything work as expected.

**Comment by thepurple** [comment]
This is a fractional scaling with scales like 125%, 150%, 175%. It should be fine on 100% and 200%.
Possibly it will be solved if we'll start to capture just current screen like on MacOS (also commented here: https://github.com/flameshot-org/flameshot/issues/1610#issuecomment-883524218).
